core/account.py

```diff
@@ -28,7 +28,7 @@ def get_account(
     """
 
     if account_id:
-        return check_exists_then_get(account_id)
+        return check_account_exists_then_return(account_id)
     elif email_address:
         return get_data_by_email(email_address)
```

Comment:
I know not part of this PR but the finicky part of me says we could change this function name to get_account_data_by_email 👀

Reply:
also typo on line 35 (again not your error)